.gitignore

```diff
@@ -0,0 +1,11 @@
+/target/
+/checkouts/
+.lein-deps-sum
+.lein-repl-history
+.lein-plugins/
+.lein-failures
+.nrepl-port
+.cpcache/
+/tmp/
+.DS_Store
+
```

Comment:
I also have these three VSCode-specific lines:
```
.calva
.clj-kondo/.cache
.lsp
```
Not really needed if nobody with VSCode (i.e. me) will touch the code but would be nice.

Reply:
done